mvvm-flow/apple/xcode/mokoMvvmFlowSwiftUI/ViewModelStateNullable.swift

```diff
@@ -95,14 +95,6 @@ extension ObservableObject where Self: ViewModel {
             mapper: { $0 as? Array<T> }
         )
     }
-    
-    func stateNullable(_ flowKey: KeyPath<Self, CStateFlow<StringDesc>>) -> String? {
-        return stateNullable(
-            flowKey,
-            equals: { $0 === $1 },
-            mapper: { $0?.localized() }
-        )
-    }
```

Comment:
it's should not be deleted. it's should be moved to mvvm-flow-resources

Reply:
No problem, I'll make that change.

Reply:
Hmm, there isn't a separate mvvm-flow-resources SwiftUI project (something like `mokoMvvmFlowResourcesSwiftUI`) to move this particular piece of code to.